### DISCUSSION TITLE
How to contribute?

### DISCUSSION THREAD

**Discussion by MalcolmReed-ent** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#771] [Q&A]
I've just noticed that this is a continuation of the original project a couple of days ago,  and there are some things, like MediaTek and a few other packages, that need descriptions in general. 

I'm thinking of looking up each package to see if it's important, then maybe disabling them on my phone to test things out. After that, I can document what I find and put together a pull request. Any tips on how to go about this?

Is that the path that should be taking in writing a description for a package.

**Comment by AnonymousWP** [comment]
Yes, for sure! See here how to contribute: https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/wiki/How-to-contribute.

**Comment by MalcolmReed-ent** [comment]
Thank you. I'm planning to stay engaged and contributing.
